v0/util/index.js

```diff
@@ -399,29 +399,121 @@ const updatePayload = (currentKey, eventMappingArr, value, payload) => {
   return payload;
 };
 
-const getValueFromMessage = (message, sourceKey) => {
-  if (Array.isArray(sourceKey) && sourceKey.length > 0) {
-    if (sourceKey.length === 1) {
+// This method handles the operations between two keys from the sourceKeys. One of the possible
+// usecase is to calculate the value from the "price" and "quantity"
+//
+// Definition of the operation object is as follows
+//
+// {
+//   "operation": "multiplication",
+//   "args": [
+//     {
+//       "sourceKeys": "properties.price"
+//     },
+//     {
+//       "sourceKeys": "properties.quantity",
+//       "defaultVal": 1
+//     }
+//   ]
+// }
+//
+// Supported operations are "addition", "multiplication"
+//
+const handleSourceKeysOperation = ({ message, operationObject }) => {
+  const { operation, args } = operationObject;
+
+  // populate the values from the arguments
+  // in the same order it is populated
+  const argValues = args.map(arg => {
+    const { sourceKeys, defaultVal } = arg;
+    const val = get(message, sourceKeys);
+    if (val || val === false || val === 0) {
```

Comment:
If we looking to only support arithmetic operations, I'd suggest we block `false` as well.